cli_tools/common/image/importer/errors.go

```diff
@@ -30,3 +30,14 @@ func customizeErrorToDetectionResults(osID, detectedDistro, detectedMajor,
 	}
 	return original
 }
+
+func customizeErrorToDetectionResults2(osID string, detected distro.Release, original error) error {
```

Comment:
If the error message is being improved with usage from `distro`, why will we be using the previous `customizeErrorToDetectionResults` function?

Reply:
Good find!  I didn't finish the refactoring here prior to submitting the PR.

The current version removes the old implementation and updates the tests.